packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -336,6 +347,8 @@ export class DashboardModel {
                 order,
                 views,
                 first_viewed_at,
+                validation_error,
```

Comment:
This order by will give you only the last error for this dashboard_uuid, right ?